package.json

```diff
@@ -32,7 +32,17 @@
         "lodash": "^4",
         "prettier": "^2.8.8"
     },
-    "bundleDependencies": true,
+    "bundleDependencies": [
+        "@kapeta/codegen-target",
+        "@kapeta/kaplang-core",
+        "@kapeta/nodejs-process",
+        "@kapeta/nodejs-utils",
+        "@kapeta/schemas",
+        "@kapeta/ui-web-types",
+        "@types/lodash",
+        "lodash",
+        "prettier"
+    ],
```

Comment:
I think this is equivalent to `"bundleDependencies": true`

Reply:
Oh yea I don't know why that made it in there tbh 🤔